elife.cfg

```diff
@@ -1,6 +1,7 @@
 [general]
 env: dev
+#upload_path uploads
+#cache_path: cache
 
 [lax]
 location: /srv/lax/
-cache_path: /home/elife
```

Comment:
Also `/var/cache/` or similar would be fine: the important thing is for the cache to be kept by default outside of the project folder, which is created from scratch for every new pull request